thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/FetchAnomaliesInRangeAndOutputCSV.java

```diff
@@ -0,0 +1,205 @@
+      List<FetchMetricDataAndExistingAnomaliesTool.ResultNode> resultNodes){
+    Set<String> functionIdDimension = new HashSet<>();
+    Map<String, Map<String, Double>> functionIdDimension_VS_Date_Severity = new HashMap<>();
+
+    LOG.info("Loading date-dimension anomaly results from db...");
+
+    for (FetchMetricDataAndExistingAnomaliesTool.ResultNode n : resultNodes){
+      String key = n.functionId + "," + n.dimensionString();
+      String anomalyStartTime = dateTimeFormatter.print(n.startTime);
+      if(!functionIdDimension_VS_Date_Severity.containsKey(key)){
+        functionIdDimension_VS_Date_Severity.put(key, new HashMap<String, Double>());
+      }
+      Map<String, Double> targetMap = functionIdDimension_VS_Date_Severity.get(key);
+      targetMap.put(anomalyStartTime, n.severity);
+      functionIdDimension.add(key);
+    }
+
+    try {
+      BufferedWriter bw = new BufferedWriter(new FileWriter(outputFile));
+      List<String> schemas = new ArrayList<>(functionIdDimension);
+      Collections.sort(schemas);
+
+      LOG.info("Printing raw anomaly results from db...");
+
+      // Write Schema
+      bw.write("functionId,dimension");
+      for (DateTime curr = dataRangeStart; curr.isBefore(dataRangeEnd); curr = curr.plusDays(1)) {
+        String currDate = dateTimeFormatter.print(curr);
+        bw.write("," + currDate);
+      }
+      bw.newLine();
+
+      for (String schema : schemas) {
+        bw.write(schema);
+        Map<String, Double> targetMap = functionIdDimension_VS_Date_Severity.get(schema);
+        for (DateTime curr = dataRangeStart; curr.isBefore(dataRangeEnd); curr = curr.plusDays(1)) {
+          String currDate = dateTimeFormatter.print(curr);
+          bw.write(",");
+          if (targetMap.containsKey(currDate)) {
+            bw.write(Double.toString(targetMap.get(currDate) * 100));
+          }
+        }
+        bw.newLine();
+      }
+      bw.close();
+    }
+    catch (IOException e){
+      LOG.error("Unable to write date-dimension anomaly results to given file {}", e);
+    }
+  }
+  /**
+   * Ouput merged anomaly results for given metric and time range
+   * @param args List of arguments
+   *             0: path to persistence file
+   *             1: collection name
+   *             2: metric name
+   *             3: monitoring start time in ISO format
+   *             4: timezone code
+   *             5: monitoring length in days
+   *             6: Output path
+   */
+  public static void main(String args[]){
+    if(args.length < 7){
+      LOG.error("Insufficient number of arguments");
+      return;
+    }
+    // Put arguments in Map
+    Map<String, String> argMap = new HashMap<>();
+    argMap.put("persistenceFile", args[0]);
+    argMap.put("collectionName", args[1]);
+    argMap.put("metricName", args[2]);
+    argMap.put("monitoringTime", args[3]);
+    argMap.put("timezone", args[4]);
+    argMap.put("monitorLength", args[5]);
+    argMap.put("outputPath", args[6]);
```

Comment:
Is there a specific reason we use a hashmap to store the variables? We could just store them to a variable